fairer-iOS/fairer-iOS/Screens/Home/Home/View/WeekCalendarView/HomeWeekCalendarCollectionViewCell.swift

```diff
@@ -31,11 +30,25 @@ final class HomeWeekCalendarCollectionViewCell: BaseCollectionViewCell {
     }
     
     // MARK: - property
-
     let workDot: UIImageView = {
         let imgView = UIImageView()
         return imgView
     }()
```

Comment:
p2;
```suggestion
    let workDot = UIImageView()
```
요것도 이렇게 간단하게 나타낼 수 있을 것 같아용!

Reply:
[81bc389](https://github.com/fairer-iOS/fairer-iOS/pull/132/commits/81bc389e11f3fecc461dd33aa576dc9044c2b3e9)